### ISSUE TITLE
a11y über das Projekte & Ergebnis tabs not accessible via keyboard

### ISSUE THREAD

**Issue by maxliqd** [liqd/a4-meinberlin#5226]
**URL:** e.g. https://mein.berlin.de/projekte/machbarkeitsstudie-fur-eine-mogliche-wohnbebauun-2/
**user:** visitor
**expected behaviour:** you can reach über das Projekt and Ergebnis by tabbing
**behaviour:** No way to get there with keyboard
**important screensize:**
**device & browser:** desktop
**Comment/Question:** 

Screenshot?

**Comment by CarolingerSeilchenspringer** [comment]
It is accessible via arrows but not with tab.
